### DISCUSSION TITLE
Quartz v1.0.2.2 is not executing 30 days job

### DISCUSSION THREAD

**Discussion by amolwabalezs** [quartznet/quartznet#1794] [Q&A]
Hi there,

I am executing 30 days job using simple trigger.
When ever the job is schedule for 30 days it first get triggered on immediate first occurring sunday and then triggers as per the schedule.
Following is the method in which simple trigger is initiated -

var trigger = new SimpleTrigger(triggerName, groupName, startTimeUtc, endTimeUtc, repeatCount, repeatInterval)
            {
                MisfireInstruction = CurrentMisfireInstruction
            };

triggername = "any random name in string"
groupName = "any string"
startTimeUtc = calculate 1st occurrence of trigger
endTimeUtc = null
repeatCount = -1 //Triggers unlimited
repeatInterval = interval in DateTime format after which the job is to be triggerred
CurrentMisfireInstruction = 5

Weird behavior - It gets triggerred on immediate first sunday than it get scheduled as per interval and works properly.
Question - Why does it gets triggered on immediate sunday ?

**Comment by drusellers** [comment]
Can you provide a github repository with an example reproduction? It will be much easier to pull down and run the code to see what's going on.
